README.md

```diff
@@ -108,6 +110,7 @@ myGraph(<myDOMElement>)
 | Method | Description | Default |
 | --- | --- | :--: |
 | <b>stopAnimation</b>() | Stops the rendering cycle of the component, effectively freezing the current view and canceling all future user interaction. This method can be used to save performance in circumstances when a static image is sufficient. | |
+| <b>startAnimation</b>() | Starts the rendering cycle of the component, effectively unfreezing the current view and enabling all future user interaction. This method can be used to save performance together with `stopAnimation` method. | |
```

Comment:
Thanks for this PR @artyomtrityak, and sorry for the slow responses. What do you think about calling this method `resumeAnimation` instead, since it's only applicable after it's been stopped programmatically?